CefSharp/Internals/JavascriptObjectRepository.cs

```diff
@@ -211,15 +213,31 @@ private void AnalyseObjectForBinding(JavascriptObject obj, bool analyseMethods,
                     var jsObject = CreateJavascriptObject(camelCaseJavascriptNames);
                     jsObject.Name = propertyInfo.Name;
                     jsObject.JavascriptName = GetJavascriptName(propertyInfo.Name, camelCaseJavascriptNames);
-                    jsObject.Value = jsProperty.GetValue(obj.Value);
+                    jsObject.LateBinding = () =>
+                    {
+                        if (!propertyInfo.PropertyType.IsArray)
```

Comment:
Switch logic in `if` statement, preferable to eval  `==` rather than `!=`, reads just that little bit easier.